ocean_provider/run.py

```diff
@@ -25,11 +25,27 @@ def get_version():
 
 @app.route("/")
 def version():
+    """
+    Created a global dictionary
+    to store the services' routes.
+    It is open for modifications, by simply adding the service
+    name and the url for it.    """
+
     info = dict()
     info['software'] = Metadata.TITLE
     info['version'] = get_version()
     info['network-url'] = config.network_url
     info['provider-address'] = get_provider_wallet().address
+    # Added the expose endpoint. Here are the URLs to the
```

Comment:
I think this comment is unnecessary and also relates more to your changes than the general intent of the code. If the tests change, nobody will know that this is referenced here and it will result in inconsistencies with the code.

The code is very good and self-descriptive, it does not need commenting. But if you want to leave a comment, you can say something about the intention, e.g. "# URLs to the existing endpoints in services"